cucumber/api/features/secrets_batch.feature

```diff
@@ -91,3 +91,15 @@ Feature: Batch retrieval of secrets
     { "cucumber:variable:secret1": "v3", "cucumber:variable:secret2": "v5", "cucumber:variable:secret3": "v6" }
     """
 
+  Scenario: Returns the correct result for binary secrets
+    Given I create a binary secret value for resource "cucumber:variable:secret3"
+    And I set annotation "conjur/mime_type" to "application/octet-stream"
```

Comment:
is this annotation set on the secret in the DB?

Reply:
Yes, it adds the annotation to the secret in the database